azure-monitor-opentelemetry-distro/README.md

```diff
@@ -28,6 +28,18 @@ Install the Azure Monitor Opentelemetry Distro with [pip][pip]:
 pip install azure-monitor-opentelemetry-distro --pre
 ```
 
+### Manual Configuration
+
+You can use the `configure_opentelemetry` to set up manual instrumentation in a single line and send telemetry for your app to Azure Monitor. `configure_opentelemetry` supports the following optional arguments:
+
+* connection_string - You can pass in your connections string here. If left out, the [Azure Monitor OpenTelemetry exporters][azure_monitor_opentelemetry_exporters] will use the connection string specified in the `APPLICATIONINSIGHTS_CONNECTION_STRING` environment variable. 
```

Comment:
As part of these definitions, it would be great if we could link to some external documentation referencing to what these things are. For example:

[logging_level][logging_level_doc]

[logging_level_doc]: https://docs.python.org/3/library/logging.html#levels

The service related ones can be found [here](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions#service)

Reply:
Done